Pipfile

```diff
@@ -1,23 +1,21 @@
 [[source]]
-
 url = "https://pypi.python.org/simple"
 verify_ssl = true
 name = "pypi"
 
-
 [packages]
-
 certifi = "*"
 chardet = "*"
 idna = "*"
 requests = ">=2.20.0"
-urllib3 = ">=1.23"
-
+urllib3 = ">=1.24.2"
```

Comment:
Please set the version of all dependencies in order to prevent installation errors due to  incompatible vendor releases.

Those are the dev dependencies version which should work:
```
[dev-packages]
pytest = "==3.6.0"
"pytest-mock" = "==1.10.0"
"pytest-cov" = "==2.5.1"
isort = "*"
pylint = "==1.9.1"
```